packages/rendermime/src/factories.ts

```diff
@@ -20,7 +20,7 @@ export const htmlRendererFactory: IRenderMime.IRendererFactory = {
  */
 export const imageRendererFactory: IRenderMime.IRendererFactory = {
   safe: true,
-  mimeTypes: ['image/bmp', 'image/png', 'image/jpeg', 'image/gif'],
+  mimeTypes: ['image/bmp', 'image/png', 'image/jpeg', 'image/gif', 'image/webp'],
```

Comment:
Prettier is not happy :wink: 

```suggestion
  mimeTypes: [
    'image/bmp',
    'image/png',
    'image/jpeg',
    'image/gif',
    'image/webp'
  ],
```

Reply:
I have updated the test at packages/rendermime/test/factories.spec.ts:L428, and I have also resolved the Prettier error is also resolved now. @fcollonval